main.go

```diff
@@ -83,6 +87,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := triggersapi.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
Just some thoughts here, not really a review:

So this introduces a dependency on Tekton and OpenShift or HAS. I don't think we can necessarily avoid it this time (for obvious reasons, as they're needed by these changes), but being able to dev/test HAS against something considerably lighter-weight than CRC (e.g. kcp, k3s, kind, minikube) has been nice.

Wonder if [microshift](https://github.com/redhat-et/microshift) + tekton would meet the requirements for what I'm looking for in local dev? Or if there's some way we could mock routes + tekton on something like kcp/kind (like just install the CRD, but not the corresponding controller?)

Reply:
( getting back to this to share my thoughts shortly )